### DISCUSSION TITLE
Creating questions individually

### DISCUSSION THREAD

**Discussion by vaivk369** [Sunbird-inQuiry/Community#103] [Feature Requests]
Can we have a way where we can create questions outside the Question set? We need to maintain the question bank or question library, create questions of various types and then while creating question sets we can choose questions from the library or create new.
Adding questions from library while creating question set is there, but need a way to create these questions individually.

@PoojaJManjooran

**Comment by PoojaJManjooran** [comment]
Yes @vaivk369 - The question create / edit APIs can be used to create questions which are not part of any question set. @rajnishdargan - You can give more info on the editor capabilities for the same.
